css/grid.css

```diff
@@ -0,0 +1,28 @@
+.container{
+    min-height: 100vh;
+    display: grid;
+    gap: 3px;
+    grid-template-rows: auto auto auto 1fr;
+    grid-template-areas: 
+        "header"
+				"navbar"
```

Comment:
Would you mind, removing this whitespace on the left of "navbar"

Reply:
Done